tests/output/runner.rs

```diff
@@ -1232,6 +1233,80 @@ async fn test_step_with_measurement_series_element_with_metadata_index_no() -> R
     .await
 }
 
+#[tokio::test]
```

Comment:
can you split this into a different file? `diagnosis.rs`

splitting runner.rs as a whole is on my todo list, because the `ocptv::output` module is already split, and this file is too huge now